pysumma/preprocessing/input_netcdf.py

```diff
@@ -0,0 +1,138 @@
+import pandas as pd
+from netCDF4 import Dataset, num2date, date2num
+import numpy as np
+from datetime import datetime, timedelta
+
+
+def InitialCond_from_csv(hru_path, ifcToto_path, midToto_path, name):
+    """Create initial conditons netcdf from csv file
+
+    Args:
+        hru_path (csv file): A CSV file with variables using hru Dimensions
+        ifcToto_path (csv file): A CSV file with variables using ifcToto Dimensions
+        midToto_path (csv file): tA CSV file with variables using midToto Dimensions
+        name (text): Set the name of netcdf (ex. initial_conditions.nc)
+    """
+
+    # Create a netCDF file
+    initial_cond = Dataset(name, "w", format="NETCDF3_CLASSIC")
```

Comment:
I'd prefer to use `xarray` here just to be internally consistent. Also, is there a reason you're using `NETCDF3_CLASSIC`? If it's not a hard constraint I'd rather use something either newer, or (even better IMO) agnostic to the version.